src/main/webapp/view/transcript/transcript-area.js

```diff
@@ -71,15 +71,18 @@ export default class TranscriptArea {
    * <p>This is a private method that should only be called in
    * `loadTranscript()`.
    */
-  static addTranscriptLinesToDom(transcriptLines) {
+  addTranscriptLinesToDom(transcriptLines) {
     const transcriptContainer = TranscriptArea.transcriptScrollContainer();
     const ulElement = document.createElement('ul');
     // TODO: Move the class assignment to the HTML.
     ulElement.class = 'mx-auto';
     transcriptContainer.appendChild(ulElement);
     transcriptLines.forEach((transcriptLine) => {
-      ulElement.appendChild(
-          TranscriptLineElement.createTranscriptLineElement(transcriptLine));
+      const transcriptLineElement =
+          TranscriptLineElement.createTranscriptLineElement(transcriptLine);
```

Comment:
Does this do any work?

Reply:
It calls the constructor for creating a `transcriptLineElement` [link to code](https://github.com/googleinterns/zoomtube/blob/master/src/main/webapp/view/transcript/transcript.js#L46)

Reply:
For some reason I thought this was part of the constructor.